src/istep/help/istep_list.htxt

```diff
@@ -0,0 +1,2 @@
+istep names for Power Systems and respective step number
```

Comment:
This file is added here because the openbmc recipe which pulls this file will apply a patch over it and update istep names. These istep names will be specific to the platform

Reply:
Why do we need an essentially empty file in the repo for another process to use?  Why can't that recipe just straight copy over the file?

Reply:
There needs to be some base file to apply the patch over it. else, we need to have this in some repo

Reply:
> Why do we need an essentially empty file in the repo for another process to use? 

Right, I don't think we should do that either.

> Why can't that recipe just straight copy over the file?

Well it could, but that is typically the job of the build system (e.g. configure/build/install).  If I run "make install" when I configure/build/install this package shouldn't it produce something that works?

At the end of the day, regardless of whether or not the help text is platform specific, this code still depends on it (do I have that right?) does it not?

The usual way of handling this is to make the user (the person running configure/build/install) provide the platform specific file in the configure step, and then simply install the provided file in the install step (basically doing what @thejta suggested but in the build process of this package instead of in the recipe).  Thoughts?